util/zip-exact/src/lib.rs

```diff
@@ -0,0 +1,34 @@
+#![no_std]
+
+use core::{
+    fmt::{self, Debug, Display},
+    iter::Zip,
+};
+use serde::{Deserialize, Serialize};
```

Comment:
Basic question: Why do we need De/Serialize?

Reply:
I had to add it because `RingSignatureError` implements these. I think it's because it can get passed across the enclave boundary as part of a `TxValidationError`.